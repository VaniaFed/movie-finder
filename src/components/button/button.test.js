import { shallow } from 'enzyme';
import { Button } from './index';

describe('<Button />', () => {
    const component = shallow(<Button>Hello, React!</Button>);
    expect(
        component.find('button.search-block__btn-search button').length
    ).toBe(5);
});
