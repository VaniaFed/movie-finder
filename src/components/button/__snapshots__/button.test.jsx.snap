// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> should be rendered with text "Hello, React!" 1`] = `
<button
  className="button"
  type="button"
>
  Hello, React!
</button>
`;
